cmd/analyzer/main.go

```diff
@@ -83,6 +84,16 @@ func vpcConfigsFromFile(fileName string, inArgs *InArgs) (map[string]*vpcmodel.V
 	return vpcConfigs, nil
 }
 
+func translateCDtoConnectionSet(inArgs *InArgs) *common.ConnectionSet {
+	connection := common.NewConnectionSet(false)
+	if inArgs.Protocol == "icmp" {
+		connection.AddICMPConnection(inArgs.SrcMinPort, inArgs.SrcMaxPort, common.MinPort, common.MaxPort)
```

Comment:
Please have the `portMin` and `portMax` for `src` and `dst` be potential part of the input; if it is left unspecified than use the `min` and `max` values as you do now.